{
  "errors": [
    {
      "message": "There is already a model with the name Person",
      "hint": "Use unique names between models, enums and messages",
      "code": "NamingError",
      "pos": {
        "filename": "testdata/validation_unique_model_names/schema.keel",
        "offset": 25,
        "line": 5,
        "column": 7
      },
      "endPos": {
        "filename": "testdata/validation_unique_model_names/schema.keel",
        "offset": 31,
        "line": 5,
        "column": 13
      }
    },
    {
      "message": "There is already a model with the name Person",
      "hint": "Use unique names between models, enums and messages",
      "code": "NamingError",
      "pos": {
        "filename": "testdata/validation_unique_model_names/schema.keel",
        "offset": 47,
        "line": 9,
        "column": 6
      },
      "endPos": {
        "filename": "testdata/validation_unique_model_names/schema.keel",
        "offset": 53,
        "line": 9,
        "column": 12
      }
    },
    {
      "message": "There is already a model with the name Person",
      "hint": "Use unique names between models, enums and messages",
      "code": "NamingError",
      "pos": {
        "filename": "testdata/validation_unique_model_names/schema.keel",
        "offset": 68,
        "line": 13,
        "column": 9
      },
      "endPos": {
        "filename": "testdata/validation_unique_model_names/schema.keel",
        "offset": 74,
        "line": 13,
        "column": 15
      }
    }
  ]
}